modules/core/src/smithy4s/http/internals/HttpResponseCodeSchemaVisitor.scala

```diff
@@ -49,6 +40,15 @@ class HttpResponseCodeSchemaVisitor()
     case _ => NoResponseCode
   }
 
+
+  override def enumeration[E](shapeId: ShapeId, hints: Hints, values: List[EnumValue[E]], total: E => EnumValue[E]): ResponseCodeExtractor[E] =
```

Comment:
**issue:** this should ideally be covered by a test, even if the implementation is trivial it gives us safety that we won't accidentally break this in the future